api/api/models/vulns.py

```diff
@@ -21,34 +19,48 @@ class Notes(models.Model):
     content: models.TextField = models.TextField(max_length=MAX_NOTE_LENGTH)
     creation_date: models.DateTimeField = models.DateTimeField(auto_now_add=True, editable=False)
     last_updated_date: models.DateTimeField = models.DateTimeField(auto_now_add=True, editable=True)
-    author: models.ForeignKey = models.ForeignKey(Pentester, on_delete=models.CASCADE)
+    author: models.ForeignKey = models.ForeignKey(Auth, on_delete=models.CASCADE, blank=True, null=True)
 
 
 class ImageModel(models.Model):
     class Meta:
         verbose_name = 'Image Model'
         verbose_name_plural = 'Image models'
         ordering = []
-    image = ImageField(name='image') # FIXME(adina): add storage, STATIC_FILES path in settings, setup nginx
+
+    image = ImageField(name='image')  # FIXME(adina): add storage, STATIC_FILES path in settings, setup nginx
+
 
 class VulnType(models.Model):
     class Meta:
         verbose_name = 'Vulnerability TYPE Model'
         verbose_name_plural = 'Vulnerability TYPES models'
         ordering = ['name']
+
     name = models.CharField(max_length=NAME_LENGTH)
     description = models.TextField(blank=True)
 
+    def __str__(self):
+        return self.name
+    def __repr__(self):
+        return f'<VulnType: \'{self.name}\'>'
+
+
 class Vulnerability(models.Model):
     class Meta:
         verbose_name = 'Vulnerability Model'
         verbose_name_plural = 'Vulnerability models'
         ordering = ['creation_date']
+
     title = models.CharField(max_length=NAME_LENGTH)
     description = models.TextField(max_length=MAX_NOTE_LENGTH, blank=True)
+
     creation_date: models.DateTimeField = models.DateTimeField(auto_now_add=True, editable=False)
     last_updated_date: models.DateTimeField = models.DateTimeField(auto_now_add=True, editable=True)
-    author: models.ForeignKey = models.ForeignKey(Pentester, on_delete=models.CASCADE, related_name='author')
-    last_editor = models.ForeignKey(Pentester, on_delete=models.CASCADE, related_name='last_editor')
-    vuln_type = models.OneToOneField(VulnType, on_delete=models.CASCADE)
-    images = models.ManyToManyField(ImageModel)
\ No newline at end of file
+
+    author: models.ForeignKey = models.ForeignKey(
```

Comment:
type hint could be `Optional[Auth]` here too ? idk if it's very important but if it makes the code cleaner, it's nice